gui_qt/app.py

```
@@ -296,11 +299,53 @@ def __init__(
         QTimer.singleShot(0, self._deferred_startup_refresh)
         QTimer.singleShot(0, self._sync_work_mode_hint_height)
 
+        # Keyboard shortcuts
+        self._setup_shortcuts()
+
         # Status
         self.statusBar().showMessage(
             "图形界面是可选组件；核心命令行不受影响。"
```

Ruff (0.15.20)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)